src/components/BoxSelection/BoxSelection.driver.ts

```diff
@@ -0,0 +1,48 @@
+import {
+  BaseUniDriver,
+  baseUniDriverFactory,
+} from 'wix-ui-test-utils/base-driver';
+import { UniDriver } from 'wix-ui-test-utils/unidriver';
+import { BOX_SELECTION_DATA_HOOKS } from './dataHooks';
+
+export interface BoxSelectionDriver extends BaseUniDriver {
+  isChecked(id: string): Promise<boolean>;
+  isDisabled(id: string): Promise<boolean>;
+  isUnavailable(id: string): Promise<boolean>;
+  getOptionsCount(): Promise<number>;
+  clickOnOption(id: string): Promise<void>;
+}
+export const boxSelectionDriverFactory = (
+  base: UniDriver,
+): BoxSelectionDriver => {
+  const boxOptionDataHook = `[data-hook=${BOX_SELECTION_DATA_HOOKS.BOX_SELECTION_OPTION}]`;
+  const getOptionById = async (id: string) => base.$(`[id="${id}"]`);
+  const getOptionByDataId = async (id: string) => base.$(`[data-id="${id}"]`);
```

Comment:
yes because **id** is for the native radio button and **data-id** is for the div that wraps the radio.
I needed the **data-id** because I need to get the '**unavailable**' attribute.
since it does not exist on the native.